test/src/integration/transactions.test.ts

```diff
@@ -191,6 +191,50 @@ describe("transactions", () => {
         expect(await node.sdk.rpc.chain.getAsset(tx2.hash(), 0)).toBe(null);
     });
 
+    // FIXME: An error occurred while chain_getParcelInvoice: FetchError:
```

Comment:
https://github.com/CodeChain-io/codechain/commit/814e986cc46c7340381e9ecc12872162ee17aece fixes it